packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -137,15 +143,6 @@ export interface IMoveInfo {
 	 */
 	moveDst?: ReferencePosition;
 
-	/**
-	 * List of client IDs that have moved this segment.
-	 *
-	 * The client that actually moved the segment (i.e. whose move op was sequenced
-	 * first) is stored as the first client in this list. Other clients in the
-	 * list have all issued concurrent ops to move the segment.
-	 */
-	movedClientIds: number[];
```

Comment:
Note that this constitutes a breaking (legacy) API change. I believe we are still not allowing such changes, until Loop has caught up with our previous releases.

Reply:
Yes, the plan is to hold off on merging this until the breaking release.